src/js/slide.js

```diff
@@ -0,0 +1,74 @@
+//$(document).ready(function() {
+
+
+/*
+$('#slideshow').on('submit' , function(elem) {
+  elem.preventDefault();
+  var input = $('#inputspot').val();
+
+  $.get('http://reddit.com/search.json', {
+    q: '#inputspot'
+  }).done(function(data) {
+    console.log('done!');
+});
+
+.fail(function(data) {
+  console.log('search failed');
+})
+}
+*/
+
+//Set up the page when it loads
+$(function() {
+// attach the form submission to the search function
+$("#search-form").on('submit', search);
+});
+
+function search(event) {
+  //stop the form from changing the page.
+  event.preventDefault();
+
+  clearSearchResults();
+
+  //get the users search input and save it in a variable.
+  //use the input placeholder value (like "kittens") as default value
+  var input = $("#query");
+  var useQuery = input.val() || input.attr("placeholder");
+
+  console.log("searching for:", useQuery);
+
+  $.get('https://www.reddit.com/search.json', {
+    q: userQuery
```

Comment:
This variable doesn't match the one you defined above, so it'll throw an error for an undefined value.